extensions/amp-gwd-animation/0.1/amp-gwd-animation-impl.js

```diff
@@ -71,6 +71,13 @@ const VENDOR_ANIMATIONEND_EVENTS = ['animationend', 'webkitAnimationEnd'];
  */
 const GOTO_AND_PAUSE_DELAY = 40;
 
+/**
+ * Property name used to store pending goto counters on an element.
+ * @const {string}
+ * Exported for test only.
+ */
+export const GOTO_COUNTER_PROP = 'gwdGotoCounters';
```

Comment:
Nit: We aim to use element properties defined under `__AMP_THIS_FORMAT__`. Could you use `__AMP_GWD_GOTO_COUNTERS__`?

Reply:
Done.